tools/cli/workflow_commands.go

```diff
@@ -295,22 +298,22 @@ func startWorkflowHelper(c *cli.Context, shouldPrintProgress bool) {
 		table.Render()
 
 		printWorkflowProgress(c, domain, wid, resp.GetRunID())
+		return nil
 	}
 
 	if shouldPrintProgress {
-		runFn()
-	} else {
-		startFn()
+		return runFn()
 	}
+	return startFn()
 }
 
-func constructStartWorkflowRequest(c *cli.Context) *types.StartWorkflowExecutionRequest {
-	domain := getRequiredGlobalOption(c, FlagDomain)
+func constructStartWorkflowRequest(c *cli.Context) (*types.StartWorkflowExecutionRequest, error) {
+	domain := getRequiredOption(c, FlagDomain)
 	taskList := getRequiredOption(c, FlagTaskList)
 	workflowType := getRequiredOption(c, FlagWorkflowType)
 	et := c.Int(FlagExecutionTimeout)
 	if et == 0 {
-		ErrorAndExit(fmt.Sprintf("Option %s format is invalid.", FlagExecutionTimeout), nil)
+		ErrorAndPrint(fmt.Sprintf("Option %s format is invalid.", FlagExecutionTimeout), nil)
```

Comment:
this might be the reason for that mock change, as it doesn't short circuit any more

Reply:
fixed all other such cases from errcheck